server/cmd/dcrdex/main.go

```diff
@@ -97,8 +98,30 @@ func mainCore(ctx context.Context) error {
 		return err
 	}
 
+	adminWebServerDone := make(chan struct{})
```

Comment:
We've been pretty consistent about using `WaitGroup` for this purpose, and it eliminates the need for the `else` block to close the channel.